openlibrary/templates/books/carousel_item.html

```diff
@@ -26,36 +25,18 @@
                 </div>
             </div>
         </a>
-
-    $if book.get("read_url"):
+    $if book.get("read_url") or loans:
         <div class="coverEbook">
-            <a href="$book.read_url?ref=ol" $:(('data-ol-link-track="%s"' % pixel) if pixel else '') title="$_('Read online')">
-              <img src="/images/icons/icon_ebook-avail.png" border="0" width="32" height="33" alt="$_('Read online')"/>
+            <a href="$((book.get('inlibrary_borrow_url') or book.get('borrow_url')) if loans else book.read_url)?ref=ol" $:(('data-ol-link-track="%s"' % pixel) if pixel else '') title="$_('Read online')" class="borrow_available cta-btn">
+              Read
             </a>
         </div>
-    $elif book.get("borrow_url"):
+    $else:
         <div class="coverEbook">
-          <span class="actions read">
-            <a href="$book.borrow_url" title="$_('Read this book')" class="borrow-link"
+            <a href="$(book.get('inlibrary_borrow_url') or book.get('borrow_url'))" title="$_('Borrow this book')" class="borrow-link"
```

Comment:
`book.get('inlibrary_borrow_url') or book.get('borrow_url')` is also used up above. Seems brittle. What about extracting this out to a well-named helper variable, and using the variable in these locations instead?